Casks/cilicon.rb

```diff
@@ -0,0 +1,24 @@
+cask "cilicon" do
+  version "2.1.0"
+  sha256 "af39cff356afb5c22f006349a72f0bf1db02ad55bb2401eeb6bd360308fd9c01"
+
+  url "https://github.com/traderepublic/Cilicon/releases/download/v#{version}/Cilicon.zip"
+  name "cilicon"
+  desc "Self-Hosted ephemeral CI on Apple Silicon"
+  homepage "https://github.com/traderepublic/Cilicon"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
+  depends_on macos: :ventura
```

Comment:
I can't find this requirement anywhere upstream

Reply:
@Marcocanc At the moment you are specifying that this runs only on Ventura. If Ventura is a minimum then the stanza should be adjusted to;

```suggestion ruby
  depends_on macos: ">= :ventura"
```